meteor/client/ui/RundownList.tsx

```diff
@@ -298,14 +298,13 @@ export const RundownList = translateWithTracker(() => {
 												</Tooltip>
 											</span>
 											{/* <span className="rundown-list-item__problems">{t('Problems')}</span> */}
-											<span>{t('Show style')}</span>
+											<span>{t('Show Style')}</span>
```

Comment:
Will this invalidate the translation keys, and thus also the existing translation?

Reply:
Yeah... "Such Is Life" 🇱🇻